disclosures/2023-05-21.md

```diff
@@ -0,0 +1,60 @@
+# Incident disclosure - 2023-05-21
+
+## Summary
+- Yearn's yVault staking zap on Optimism did not properly refund a user attempting to zap in funds above the USDT vault's deposit limit. Excess funds sent to the zap should have been returned to the user, but instead they remained in the zap contract.
```

Comment:
would change this to:

- Yearn's yVault staking zap on Optimism did not properly handle an edgecase when a user deposit brings the vault's total assets above its specified "deposit limit". 
- As a result, only the allowable portion of the deposit was successfully zapped, while any excess funds remain in the zap contract